homeassistant/components/apcupsd/sensor.py

```diff
@@ -34,392 +34,343 @@
     "alarmdel": SensorEntityDescription(
         key="alarmdel",
         translation_key="alarm_delay",
-        name="UPS Alarm Delay",
     ),
     "ambtemp": SensorEntityDescription(
         key="ambtemp",
-        name="UPS Ambient Temperature",
+        translation_key="ambient_temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "apc": SensorEntityDescription(
         key="apc",
         translation_key="apc_status",
-        name="UPS Status Data",
         entity_registry_enabled_default=False,
     ),
     "apcmodel": SensorEntityDescription(
         key="apcmodel",
         translation_key="apc_model",
-        name="UPS Model",
         entity_registry_enabled_default=False,
     ),
     "badbatts": SensorEntityDescription(
         key="badbatts",
         translation_key="bad_batteries",
-        name="UPS Bad Batteries",
     ),
     "battdate": SensorEntityDescription(
         key="battdate",
         translation_key="battery_replacement_date",
-        name="UPS Battery Replaced",
     ),
     "battstat": SensorEntityDescription(
         key="battstat",
         translation_key="battery_status",
-        name="UPS Battery Status",
     ),
     "battv": SensorEntityDescription(
         key="battv",
-        name="UPS Battery Voltage",
+        translation_key="battery_voltage",
         native_unit_of_measurement=UnitOfElectricPotential.VOLT,
         device_class=SensorDeviceClass.VOLTAGE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "bcharge": SensorEntityDescription(
         key="bcharge",
-        name="UPS Battery",
+        translation_key="battery",
         native_unit_of_measurement=PERCENTAGE,
         device_class=SensorDeviceClass.BATTERY,
         state_class=SensorStateClass.MEASUREMENT,
     ),
     "cable": SensorEntityDescription(
         key="cable",
         translation_key="cable_type",
-        name="UPS Cable Type",
         entity_registry_enabled_default=False,
     ),
```

Comment:
What are the possible values?

(enum candidate)

Reply:
```
# UPSCABLE <cable>
#   Defines the type of cable connecting the UPS to your computer.
#
#   Possible generic choices for <cable> are:
#     simple, smart, ether, usb
#
#   Or a specific cable model number may be used:
#     940-0119A, 940-0127A, 940-0128A, 940-0020B,
#     940-0020C, 940-0023A, 940-0024B, 940-0024C,
#     940-1524C, 940-0024G, 940-0095A, 940-0095B,
#     940-0095C, 940-0625A, M-04-02-2000
#
```

This probably can't be enums given the fact that specific cable model numbers can be used here.

Reply:
So this will return either `simple`, `smart`, `usb` or a cable version? That's strange lol

Reply:
Yeah it sounds like it, mine returns "USB Cable" 🤦

APCUPSD really is weird in some cases (and it hasn't been updated since 2016) and the documentation isn't clear...